<?xml version="1.0"?>
<launch>

	<!-- Launch parameters -->
	<arg name="output" default="screen" /> <!-- screen/log -->
	<arg name="required" default="true" /> <!-- exit whole launch -->
	<arg name="respawn" default="false" /> <!-- restart -->
 
  <node pkg="mkz_bywire" type="mkz_bywire" name="mkz_bywire" respawn="$(arg respawn)" required="$(arg required)" output="$(arg output)">

  	<param name="max_speed"         value="30" />
    <param name="max_throttle"      value="0.45" />
    <param name="max_lat_acc"       value="3.5" />

    <param name="GPS_angle_calib"   value="+0.75" /> <!-- degree, '+' operation -->
    <param name="correct_cg_x"      value="1.0" /> <!-- Mcity MKZ:1.0    Denso:2.3 -->
    <param name="correct_cg_y"      value="0.0" /> <!-- Mcity MKZ:0.0    Denso:0.2 -->

    <param name="output_xmsgs"      value="true" />
    <param name="enable_safeguard"  value="true" />
  </node>  
</launch>
